src/action.js

```diff
@@ -69,8 +70,11 @@ async function exportSecrets() {
     defaultOptions.headers['X-Vault-Token'] = vaultToken;
     const client = got.extend(defaultOptions);
 
+    command.issue('add-mask', vaultToken);
```

Comment:
I know you used the line that already existed below but I believe that it is recommended to use [the specific functions from the toolkit to mark a variable or value as sensitive](https://github.com/actions/toolkit/tree/master/packages/core#setting-a-secret) if implementing a reusable action in javascript.

`core.setSecret(vaultToken)`

Also, dynamic sensitive values created by the action should be marked as secret as close to their instantiation as possible so they are not logged by mistake in between their creation and masking. I think it'd be preferable to move this line right after the moment the token is generated on l.69.

Reply:
Yup. If this is emulating [this line](https://github.com/hashicorp/vault-action/pull/353/files#diff-3eff2eb784c259a70aa99b331bdf2de36ef13b78240a7789e3e530e0f639f817R95) I believe it's using the manual commmand just because the `setSecret` function wasn't available at the time.